ppcp-gateway/cartflow/class-cartflows-pro-gateway-paypal-ppcp-angelleye-cc.php

```diff
@@ -258,7 +258,7 @@ public function get_purchase_units($order, $offer_product, $args) {
         $purchase_unit = array(
             'reference_id' => 'default',
             'amount' => array(
-                'currency_code' => $args['order_currency'],
+                'currency_code' => add_filter('angelleye_ppcp_woocommerce_currency', $args['order_currency']),
```

Comment:
@kcppdevelopers I feel like we should add a combined filter that passes an array ([currency_code => '', value => '']) to the filter functions, so that filter function can change the Currency code and corresponding values as per the conversion rate.
please apply that change everywhere.